Fast and Flexible GPU Quantization for Transformers.md

```diff
@@ -1,82 +1,193 @@
+While model parameters can always be quantized offline, its activations can be quantized **dynamically** (with the clipping range calculated for each activation during a forward pass) or **statically** (also offline). Static quantization involves running some forward passes, measuring the distribution of each activation in the network, and calculating the clipping range. This process is called **calibration**. For more information, see the next section.
 
-Dynamic quantization is generally more accurate but incurs a computational overhead when calibrating the scalar online. Therefore, **we** **only consider static quantization on GPU** as the scalar reduction is expensive relative to the INT8 matmul, and so can severely limit any performance gains.
+Dynamic quantization tends to be more accurate but requires additional computational overhead for online scalar calibration. As a result, **we only consider static quantization on GPU** because scalar reduction (relative to an INT8 matmul) can be costly and limit performance gains.
 
 ### Quantization Granularity
-A final distinction to be made is how we share quantization parameters between elements of our parameters and activations. For the rest of this blog, we will use the following diagram to describe a matmul:
+A final distinction to be made is how we quantization parameters are shared between elements of our parameters and activations. Throughout this blog, we'll use the following diagram to illustrate a matmul:
 
 ![](_attachments/Pasted%20image%2020230313162138.png)
 
-The simplest approach is to use the same scale factor for all elements of W (and likewise for X). This is **per-tensor** quantization.
+The simplest approach is to use the same scale factor for all elements of $W$ (and likewise for $X$). This is known as **per-tensor** quantization.
 
-It’s also feasible to share quantization parameters between some subgroups of each input matrix. A common choice is to assign a specific scale factor to each column of W. This is **per-channel (aka per-column) quantization**.
+It’s also feasible to share quantization parameters between some subgroups of each input matrix. A popular options is to assign a specific scale factor to each column of $W$, referred to as **per-channel (or per-column) quantization**.
 
-## Theory
-Having covered the basics of quantization, we’ll now look at the important concepts in their implementation.
+## Important Concepts
+With the fundamentals of quantization covered, let's explore the important concepts in its implementation.
 
 ### Calibration
```

Comment:
Think the calibration section should go before Quantization Granularity as is referenced in Dynamic vs Static Quantization

Reply:
Have reordered